projects/lib/src/dynamic-views/dynamic-views.module.ts

```diff
@@ -1,16 +1,11 @@
 import { NgModule } from '@angular/core';
-import { CommonModule } from '@angular/common';
 import { DndModule } from 'ngx-drag-drop';
 
-import { ConfigurableModule } from '../configurable/configurable.module';
-import { UtilsModule } from '../utils/utils.module';
 import { ConfigModule } from '../configuration';
-import { ZoneComponent } from './zone/zone.component';
-import { ItemComponent } from './item/item.component';
 
 @NgModule({
-  imports: [CommonModule, DndModule, ConfigurableModule, UtilsModule, ConfigModule],
-  declarations: [ZoneComponent, ItemComponent],
-  exports: [UtilsModule, ZoneComponent],
+  imports: [DndModule, ConfigModule],
```

Comment:
Do you still need this module ? It does not declare or export anything, so I don't think it's needed.

Reply:
you right, no need anymore